sql/plan.go

```diff
@@ -80,6 +80,8 @@ func (p *planner) makePlan(stmt parser.Statement) (planNode, error) {
 		return p.Select(n)
 	case *parser.Set:
 		return p.Set(n)
+	case *parser.SetTransaction:
+		return p.SetTransaction(n)
```

Comment:
I see you are changing the transaction isolation level here. I think you want to rework this to only change the session settings.